packages/frontend/src/components/chart/TokenControls.tsx

```diff
@@ -58,7 +60,7 @@ function SelectButton() {
 function SelectedTokenButton() {
   return (
     <label
-      className="flex hidden cursor-pointer items-center justify-between gap-1.5 rounded-md px-2 text-base transition-all dark:bg-black"
+      className="flex hidden cursor-pointer items-center justify-between gap-1.5 rounded-md bg-white px-2 text-base transition-all dark:bg-black"
```

Comment:
flex and hidden apply the same property (display)

Reply:
I know, but we are going to remove and add hidden to show/hide this element. If we only specified `hidden` we would have to 

1. Remove `hidden` and add `flex` for showing
2. Add `hidden` and remove `flex` for hiding

Which only adds more complexity